bundles/org.openhab.binding.amazonechocontrol/src/main/java/org/openhab/binding/amazonechocontrol/internal/Connection.java

```diff
@@ -1183,18 +1200,18 @@ public void smartHomeCommand(String entityId, String action, @Nullable String pr
                 if (errors != null && errors.isJsonArray()) {
                     JsonArray errorList = errors.getAsJsonArray();
                     if (errorList.size() > 0) {
-                        logger.info("Smart home device command failed.");
-                        logger.info("Request:");
-                        logger.info("{}", requestBody);
-                        logger.info("Answer:");
+                        logger.warn("Smart home device command failed.");
+                        logger.warn("Request:");
+                        logger.warn("{}", requestBody);
+                        logger.warn("Answer:");
                         for (JsonElement error : errorList) {
-                            logger.info("{}", error.toString());
+                            logger.warn("{}", error.toString());
                         }
                     }
                 }
             }
         } catch (URISyntaxException e) {
-            logger.info("Wrong url {}", url, e);
+            logger.warn("Wrong url {}", url, e);
```

Comment:
Please revert these changes. The upper part (l. 1186-1191) should be reworked completely. I'll provide a PR for that.

Reply:
cf. https://github.com/openhab/openhab-addons/pull/9241

Reply:
> Please revert these changes.

Reverted.